impeller/renderer/compute_tessellator.h

```diff
@@ -0,0 +1,74 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include "flutter/fml/macros.h"
+#include "impeller/geometry/path.h"
+#include "impeller/renderer/buffer_view.h"
+#include "impeller/renderer/command_buffer.h"
+#include "impeller/renderer/context.h"
+
+namespace impeller {
+
+//------------------------------------------------------------------------------
+/// @brief      A utility that generates triangles of the specified fill type
+///             given a polyline. This happens on the CPU.
```

Comment:
nit: what happens on the CPU? Polyline generation right?

Reply:
I'll fix this, it's an unintentional miss from copying and pasting.